projects/canopy/src/lib/side-nav/side-nav-content/side-nav-content.component.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  ViewEncapsulation,
+} from '@angular/core';
+
+@Component({
+  selector: 'lg-side-nav-content',
+  templateUrl: './side-nav-content.component.html',
+  styleUrls: ['./side-nav-content.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class LgSideNavContentComponent {
+  @HostBinding('class.lg-side-nav-content') class = true;
+  @HostBinding('tabindex') tabindex = 0;
```

Comment:
Is the tabindex already `0`? I just wondered why we explicitly set it to 0 here.

Reply:
This needs to be set so that when we click on the sidebar nav item, the focus is set to the content.